CMakeLists.txt

```diff
@@ -111,7 +111,8 @@ list(APPEND _SwiftFoundation_versions
 list(APPEND _SwiftFoundation_availability_names
     "FoundationPreview"
     "FoundationPredicate"
-    "FoundationPredicateRegex")
+    "FoundationPredicateRegex"
+    "FoundationSpan")
```

Comment:
Why do we need a new availability macro?

Reply:
This requires a 6.2 toolchain to get the symbols, and won't have the same range of deployment targets as any of the others, as far as I can tell.